device.yml

```diff
@@ -3,8 +3,8 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/harp-tech/reflex-generator/main/schema/device.json
 device: Olfactometer
 whoAmI: 1140
-firmwareVersion: "1.2"
-hardwareTargets: "1.0"
+firmwareVersion: "1.3"
+hardwareTargets: "1.1"
```

Comment:
Are all changes compatible with hardware 1.0 assuming that the firmware is updated to 1.3? If so, I would leave the hardwareTargets as 1.0 until we manage to solve https://github.com/harp-tech/reflex-generator/issues/42

Reply:
Ok